### PR TITLE
sample: make Python wait for response

### PR DESCRIPTION
**Pull request by pongad**

### CODE REVIEW

File: src/test/java/com/google/api/codegen/testdata/py/python_library.baseline [hunk 1]
```diff
@@ -4300,16 +4300,12 @@ def sample_get_big_book():
...
   name = client.book_path('[SHELF_ID]', '[BOOK_ID]')
 
-  response = client.get_big_book(name)
+  operation = client.get_big_book(name)
 
-  def callback(operation_future):
-      # Handle result.
-      result = operation_future.result()
+  # Wait for the operation to finish, then return the result.
+  # Or use operation.add_done_callback to avoid blocking.
```
Comment (pongad) [inline src/test/java/com/google/api/codegen/testdata/py/python_library.baseline]: @beccasaurus I think this doc can be better but I don't know how to improve it :P

**Comment by hzyi-google** [comment]
LGTM. But we may also need to make `operation` a reserved keyword for LROs to prevent users from defining a local variable with the same name?

File: src/test/java/com/google/api/codegen/testdata/py/python_library.baseline (hunk 1 above)
Comment (beccasaurus) [reply src/test/java/com/google/api/codegen/testdata/py/python_library.baseline]: How about...

```py
def sample_get_big_book():
  """Testing calling forms"""

  client = library_v1.LibraryServiceClient()

  name = client.book_path('[SHELF_ID]', '[BOOK_ID]')

  operation = client.get_big_book(name)

  print('Waiting for operation to complete...')  # ⟸
  response = operation.result()

  print(response)
```

Short 'n sweet _( 'n [stolen](https://cloud.google.com/speech-to-text/docs/async-recognize#speech-async-recognize-gcs-python) from our existing Python samples_ 😛 _)_

Comment (pongad) [reply src/test/java/com/google/api/codegen/testdata/py/python_library.baseline]: Nice

**Comment by pongad** [comment]
@hzyi-google PTAL

File: src/main/resources/com/google/api/codegen/py/standalone_sample.snip [hunk 2]
```diff
@@ -146,16 +146,11 @@
 @end
 
 @private lroSampleCode(apiMethod, sample)
-  response = {@methodCallSampleCode(apiMethod, sample.initCode)}
+  operation = {@methodCallSampleCode(apiMethod, sample.initCode)}
 
-  def callback(operation_future):
-      @# Handle result.
-      result = operation_future.result()
+  print('waiting for operation to complete...')
```
Comment (beccasaurus) [inline src/main/resources/com/google/api/codegen/py/standalone_sample.snip]: `Waiting` <-- capitalized

File: src/main/java/com/google/api/codegen/util/py/PythonNameFormatter.java [hunk 3]
```diff
@@ -229,6 +229,8 @@ public String classFileNameBase(Name name) {
               "zip",
               // "options" is not a Python keyword, but it's used as a keyword
               // argument in the methods generated by Gapic.
-              "options")
+              "options",
+              // Used by sample snippets.
+              "operation")
```
Comment (beccasaurus) [inline src/main/java/com/google/api/codegen/util/py/PythonNameFormatter.java]: Nice

**Pull request merged**